test/cli/push/orchestrator-partial/in/main/other/keboola.orchestrator/orchestrator/phases/002-phase-2/002-task-3-disabled/task.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Task 3 - disabled",
+  "enabled": false,
+  "task": {
+    "mode": "run"
+  },
+  "continueOnFailure": true
+}
```

Comment:
A tu je novy subor v `in/out` adresary (in/out su v pripade push totozne), rovnako aj v dalsom teste.